packages/design-system/src/components/OcResourceIcon/OcResourceIcon.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { shallowMount } from '@vue/test-utils'
+import { AVAILABLE_SIZES } from '../../helpers/constants'
+import { OcResourceIcon } from '..'
+
+const resources = ['file', 'folder', 'space']
+
+function getWrapper(props) {
```

Comment:
Please move the `getWrapper` function below the tests (for consistency with other tests in `web`).